### DISCUSSION TITLE
updated function for owner in  ERC721 abstract contract (building moodNft)

### DISCUSSION THREAD

**Discussion by ighodaro-daniel** [Cyfrin/foundry-full-course-cu#2181] [Ideas]
from the course  on building NFT ,patrick called the ` _isApprovedOrOwner(msg.sender, tokenId) to flipMood() function , which I discovered to be  undeclared identifier or has be updated  and believe to be owner(tokenId) function rather

**Comment by ighodaro-daniel** [comment]
you are free to verify my claims

**Reply by EngrPips** [reply]
Hello @ighodaro-daniel, check the `chronological update` section of this GitHub page. I think this change is addressed there

**Reply by ighodaro-daniel** [reply]
thanks @EngrPips

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
